JSDemos/Demos/TreeList/UsingHeaderFilter/description.md

```diff
@@ -1 +1,7 @@
-This demo shows how to&nbsp;use the header filter for filtering data. The header filter contains a&nbsp;list of&nbsp;unique column values that you can select. To&nbsp;open this list, click the filter icon in&nbsp;a&nbsp;column header. Once you select a&nbsp;value from this list and click&nbsp;OK, the filter is&nbsp;applied, and the TreeList displays the full hierarchy down to&nbsp;matching rows, but not further.
\ No newline at end of file
+A header filter allows users to select column values from a predefined set and use the selected values to filter rows. To enable this behavior, set **headerFilter**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/#visible) to **true**. This feature can be combined with the [filter row](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/UsingFilterRow).
```

Comment:
- Displaying header filter icons is not a behavior.
- All filtering demos are located in the same group. There's no need to cross-reference them.

```suggestion
A header filter allows users to select column values from a predefined set and use the selected values to filter rows. To display header filter icons, enable the **headerFilter**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/#visible) property. Users can click this icon to open a popup menu and select values by which to filter data. If this icon should be hidden for a specific column, set the column's [allowHeaderFiltering](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowHeaderFiltering) property to **false**.
```